File: pset6/mario/less/mario.py
import cs50


def main():

    while True:
        height = cs50.get_int("Height:")
        if height > 0 and height < 9:
            break
    for i in range(height):
        for j in range(height - i - 1):  # simple math: - i - 1 == -(j+1)
            print(" ", end=' ')
        for k in range(i + 1):
            print("#", end=' ')
            
        print(" ")
        

if __name__ == "__main__":
    main()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //declare the variable for height value
    int height;
    
    do 
    {
        //prompt user to get height
        height = get_int("Height: ");
    }
    //Iterate from 1 to 8
    while (height < 1 || height > 8);
    
    //Iterate til height 
    for (int i = 0 ; i < height ; i++)
    {
        //print space to right alignment
        for (int j = 0 ; j < height - i - 1 ; j++)
        {
            printf(" ");
        }
        //print hash to make pyramid
        for (int k = 0 ; k < i + 1 ; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}